### DISCUSSION TITLE
Unterstützung neue Thermostatfunktion

### DISCUSSION THREAD

**Discussion by thorsten-gehrig** [brutella/hkknx-public#292] [General]
Hallo,
seit iOS17 gibt es eine neue Thermostat-integration - in der man separate Soll-temperaturen für Heizen und Kühlen einstellen kann.
Gibt es schon einen Plan/Roadmap dies auch in HKKnx zu unterstützen?

Gruß
Thorsten Gehrig

**Comment by brutella** [comment]
Danke, dazu gibt es noch keine Roadmap. Ich werd es mir für Version 2.8.0 vornehmen.

**Comment by brutella** [comment]
Muss hier noch was ergänzen.

Für Thermostate gibt es nur eine Soll-Temperatur. Wenn die Soll-Temperatur unter der aktuellen Temperatur liegt, dass ist die Temperatur zum Kühlen. Andersrum ist die Soll-Temperatur für das Heizen.

Was es jedoch gibt, ist eine Temperaturgrenze (jeweils fürs Heizen und Kühlen) ab wann eine Thermostat zu Heizen oder Kühlen beginnen soll. Bspw. eine Heizgrenze von 19° bewirkt, dass bei unterschreiten von 19° zu Heizen begonnen wird.

Gibt es in KNX einen Heizungsaktor der diesen Parameter über eine Gruppenadressen zur Verfügung stellt?

**Reply by thorsten-gehrig** [reply]
Ob es einen KNX Aktor gibt kann ich nicht sagen - "ich" hab die Werte allerdings (danke EDOMI und PID Reglern auf dem Server hab ich was ich brauch ;-)".
Gruß
Thorsten

**Reply by brutella** [reply]
Version [2.8.0-b8](https://github.com/brutella/hkknx-public/releases/tag/2.8.0-b8) unterstützt nun die Schwellenwerte für heizen und kühlen. Bitte testen und Bescheid geben, falls es Probleme damit gibt. (Falls du Schwellenwerte zu bestehende Temperaturregler hinzufügst, kann es sein, dass die Apple Home-App oder sogar dein iPhone neu gestartet werden muss, damit die Schwellenwerte über die App eingestellt werden kann. Scheint ein Caching Problem in der Home-App zu sein.)

**Reply by brutella** [reply]
Konntest du die Schwellenwerte in HomeKit integrieren?
